scripts/lrw.sh

```diff
@@ -338,6 +338,9 @@ setup_environment() {
   info "***Installing liquidity dependencies***"
   sh -c "cd liquidity && pipenv install --dev"
 
+  info "***Setup Liquidity Provider***"
+  PIPENV_PIPFILE=liquidity/Pipfile pipenv run python liquidity/setup_env.py || fail "Fail execute setup_liquidity.py"
```

Comment:
The failure message doesn't make sense. Probably you meant something like "Liquidity service setup failed".